packages/babel-preset-env/test/fixtures/corejs3/entry-entries-modules-chrome-71/output.mjs

```diff
@@ -1,2 +1,2 @@
-import "core-js/modules/es.object.from-entries";
-import "core-js/modules/esnext.string.replace-all";
+import 'core-js/modules/es.object.from-entries';
```

Comment:
These quote changes seem unintended/unnecessary?

Reply:
It's because Babel, when possible, reuses the original quotes.
The old implementation removed this import statements and replaced with a new copy of them, while this implementation avoids replacing an import which is already ok.

Reply:
:+1: